source/install/mattermost-omnibus.rst

```diff
@@ -6,7 +6,7 @@ Overview
 
 Mattermost Omnibus is a Debian package that packages the individual components of a Mattermost deployment into a single installation process. The package leverages the ``apt`` package manager to install and keep the components of the platform updated, using a custom CLI and ansible recipes to link those components together and configure them.
 
-Mattermost Omnibus currently supports Ubuntu's ``bionic`` and ``focal`` distributions. Support for RedHat/CentOS distributions will be available in future versions.
+Mattermost Omnibus currently supports Ubuntu's ``bionic`` and ``focal`` distributions. Support for RedHat/CentOS distributions will be available in future versions. The package bundles the E0 Mattermost platform version.
```

Comment:
I don't think we refer to E0 publicly. @wiersgallak, @jasonblais, can you confirm?

A replacement could be "The package bundles the Mattermost Enterprise Edition platform version."

Reply:
Yes, "Mattermost Enterprise Edition" is what we refer to it by, but I wonder if there's a way we can word it such that it's clear it's the free version (cc @wiersgallak)

Reply:
Thanks, @jasonblais. I reworded the sentence but was considering whether "The package bundles the unlicensed Mattermost Enterprise Edition platform version." would work.

Reply:
How about `The package bundles the free, unlicensed Mattermost Enterprise Edition.`